stack-8.6.4.yaml

```diff
@@ -26,12 +30,6 @@ extra-deps:
 - temporary-1.2.1.1
 - yaml-0.8.32
 
-flags:
-  haskell-ide-engine:
-    pedantic: true
-  hie-plugin-api:
-    pedantic: true
-
```

Comment:
I would rather see these commented out, it is quite important from our CI/QA perspective to compile in pedantic mode